server.js

```diff
@@ -1,265 +1,251 @@
-"use strict"; 
-require('dotenv').load();
+'use strict'
+require('dotenv').load()
 
-var WebSocketServer = require('websocket').server;
-var http = require('http');
-var HttpDispatcher = require('httpdispatcher');
-var dispatcher = new HttpDispatcher();
-var url = require("url");
+var WebSocketServer = require('websocket').server
+var http = require('http')
+var HttpDispatcher = require('httpdispatcher')
+var dispatcher = new HttpDispatcher()
+var url = require('url')
 
-const fs = require('fs');
+const fs = require('fs')
 
-const Speech = require('@google-cloud/speech');
-
-const speech = new Speech.SpeechClient();
+const Speech = require('@google-cloud/speech')
 
+const speech = new Speech.SpeechClient()
 
 var clients = []
 
-//Create a server
-var server = http.createServer(handleRequest);
-var wsserver = http.createServer(handleRequest);
-
-process.env.APP_ID
+// Create a server
+var server = http.createServer(handleRequest)
+var wsserver = http.createServer(handleRequest)
 
-const Nexmo = require('nexmo');
+const Nexmo = require('nexmo')
 const nexmo = new Nexmo({
-  apiKey: "dummy",
-  apiSecret: "dummy",
+  apiKey: 'dummy',
+  apiSecret: 'dummy',
   applicationId: process.env.APP_ID,
-  privateKey: "./private.key"
-});
-
+  privateKey: './private.key'
```

Comment:
May want to pass in the private key path instead of having it hard-coded. This'll need to be a String to work on Heroku and not have the private key checked in.